modules/dxp/apps/scim/scim-rest-test/src/testIntegration/java/com/liferay/scim/rest/resource/v1_0/test/UserResourceTest.java

```diff
@@ -304,21 +304,34 @@ public void testPutV2User() throws Exception {
 		assertHttpResponseStatusCode(
 			404, userResource.putV2UserHttpResponse(user1.getId(), user1));
 
+		// Update an existing user provided by another SCIM client
+
 		User user2 = testDeleteV2User_addUser();
 
+		ScimTestUtil.saveSCIMClientId(
+			com.liferay.portal.kernel.model.User.class.getName(),
+			GetterUtil.getLong(user2.getId()), TestPropsValues.getCompanyId(),
+			ScimClientUtil.generateScimClientId(
+				"scim-client-test" + RandomTestUtil.randomString()));
+
+		assertHttpResponseStatusCode(
+			409, userResource.putV2UserHttpResponse(user2.getId(), user2));
+
+		User user3 = testDeleteV2User_addUser();
```

Comment:
I think we avoid creating a 3rd user here if you assert the successful update prior to changing the `scimClientId` of user2?